examples/btor_qf_ufbv.cpp

```diff
@@ -51,4 +51,4 @@ int main()
  UnorderedTermSet ua;
  s->get_unsat_assumptions(ua);
  for (Term t : ua) { cout << t << endl; }
-}
+}
```

Comment:
Github complaints if there is no newline at the end.

```suggestion
}

```